libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -114,14 +114,24 @@ public UserIdentityService(ConfigService configService, DataStore<VcStoreItem> d
         VcHelper.setConfigService(configService);
     }
 
+    public List<VcStoreItem> getVcStoreItems(String userId) {
+        return dataStore.getItems(userId);
+    }
+
+    public VcStoreItem getVcStoreItem(String userId, String criId) {
+        return dataStore.getItem(userId, criId);
+    }
```

Comment:
This isn't really in scope of the original ticket, but what do you think about moving these methods (and the delete one) into `VerifiableCredentialService`...

That way the `VerifiableCredentialService` is all about interacting with the VC Store, and the `UserIdentityService` is about performing operations on VCs (e.g. generating a user identity, checking correlation, etc.)

It'll be more effort to refactor, so up to you :)

Reply:
Done